### ISSUE TITLE
This StackOverflow post provides even better description

### ISSUE THREAD

**Issue by JranZu** [NuGet/docs.microsoft.com-nuget#1077]
This is very poorly written and hard to follow/understand.  Please see this SO question as it makes everything much clearer
https://stackoverflow.com/questions/6876732/how-do-i-get-nuget-to-install-update-all-the-packages-in-the-packages-config

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: a3b20c46-c5f1-3a2a-738d-8aa8e69cf10c
* Version Independent ID: 7055e752-859b-6106-6802-70010d0305d9
* Content: [NuGet Package Restore](https://docs.microsoft.com/en-us/nuget/consume-packages/package-restore)
* Content Source: [docs/consume-packages/Package-Restore.md](https://github.com/NuGet/docs.microsoft.com-nuget/blob/master/docs/consume-packages/Package-Restore.md)
* Product: **nuget**
* GitHub Login: @karann-msft
* Microsoft Alias: **karann**

**Comment by Mikejo5000** [comment]
Updated doc with current procedures.
